cfg.go

```diff
@@ -4,34 +4,50 @@ import (
 	"flag"
 	"log"
 	"os"
+	"strings"
 
 	"github.com/spf13/viper"
 )
 
-const envModeVar = "GOENV"
+const (
+	envModeVar      = "GOENV"
+	localConfigName = "local"
+)
 
 var configPath string
 
+func loadEnvSpecificConfig() {
+	configName, ok := os.LookupEnv(envModeVar)
+	if !ok || configName == "" {
+		configName = localConfigName
+	}
+
+	viper.SetConfigName(configName)
+	viper.AddConfigPath(configPath)
+	if err := viper.MergeInConfig(); err != nil {
+		if configName == localConfigName {
+			log.Println(localConfigName, "file not found. Using only default config")
+			return
+		}
+
+		log.Fatalln("error reading %s configuration file: %s", configName, err)
+	}
+}
+
 // Load loads the configuration file depending on the Go environment mode
-func Load() {
+func Load(serviceName string) {
 	flag.StringVar(&configPath, "config", "./config", "Configuration dir path")
 	flag.Parse()
 
-	viper.SetConfigType("toml")
```

Comment:
Should we specify `yaml` just to make it clear for new users?

Reply:
How about using a flag that defaults to `yaml`?